sdcm/sct_events/setup.py

```diff
@@ -127,6 +127,14 @@ def enable_default_filters(sct_config: SCTConfiguration):
                                     event_class=CassandraStressLogEvent.ConsistencyError,
                                     regex=r".*Authentication error on host.*Cannot achieve consistency level for cl ONE").publish()
 
+    # Change Startup failure severity to WARNING to avoid critical failure in tests.
+    # Apply only for Cloud clusters, where every failed node startup is retried by Cloud leading to the normal
+    # cluster state at the point when SCT starts to run tests.
+    if SkipPerIssues('scylladb/siren#13281', params=sct_config) and sct_config.get('use_cloud_manager'):
```

Comment:
I would actually switch the conditions, so the github call is not evaluated when we are not using cloud (which is 99% of the time)